raiden/network/transport/matrix/transport.py

```diff
@@ -594,17 +573,14 @@ def force_check_address_reachability(self, address: Address) -> AddressReachabil
         return self._address_mgr.get_address_reachability(address)
 
     def async_start_health_check(self, node_address: Address) -> None:
-        """Start healthcheck (status monitoring) for a peer
-
-        It also whitelists the address to answer invites and listen for messages
+        """
+        Start healthcheck (status monitoring) for a peer
+        also starts listening to invites and listen for messages
```

Comment:
The docstring should be updated:
Invites are accepted independently of whitelisting / healthchecking